shippable/install_source.sh

```diff
@@ -11,7 +11,7 @@ cd ..
 rm -rf ccache-${CCACHE_VERSION} ccache-${CCACHE_VERSION}.tar.bz2
 
 
-CMAKE_VERSION=3.13.1
+CMAKE_VERSION=3.15.0
```

Comment:
Probably best for this to be 3.15.6 ( I think that is the latest dot release )

Reply:
Agreed, we should use the latest stable in the series.

Reply:
Yeap, sure, I'll update this.

Reply:
Latest Ubuntu LTS comes with 16.3: https://packages.ubuntu.com/focal/cmake

Reply:
Looks like v3.15.7 based on the git tags for CMake repo.

Reply:
just make sure it works with our current tree and does not generate all kind of policy warnings that would force us to move minimum required version before we know it....

Reply:
Yeap, 3.15.7 exists in GH